lib/cartopy/mpl/slippy_image_artist.py

```diff
@@ -47,7 +47,12 @@ def draw(self, renderer, *args, **kwargs):
         if not self.get_visible():
             return
 
-        ax = self.get_axes()
+        try:
+            # matplotlib 1.5.0+
+            ax = self.axes
```

Comment:
Yes, please include this. I'm *really* tired of this warning polluting my notebooks.